tools/scoring/score/dimensions/dimension.py

```diff
@@ -13,8 +13,81 @@
 # limitations under the License.
 """Core component base class"""
 
-from score.types_ import DeserializedFilesDict, TranslationsDict
+from score.types_ import DeserializedFilesDict, TranslationsDict, PointsVirtualList, RawFieldName, EntityType, FileType
 from validate.entity_instance import EntityInstance
+from typing import Tuple, Set, List, Dict, NamedTuple
+from collections import defaultdict
+
+
+class _VirtualEntityMatch(NamedTuple):
+  """ Reference for metrics by which subscores were
```

Comment:
nit, can you please remove those empty spaces and end with a `.`
use thiis pylint style guide? https://google.github.io/styleguide/pylintrc

Reply:
I'm using this: https://github.com/google/digitalbuildings/blob/master/styles/pylintrc Is it not correct?

Reply:
https://github.com/google/digitalbuildings/pull/499/commits/68db27e44f32ce704c613c4ce70043dcf9b1ddb9